src/cartesian/Bar.tsx

```diff
@@ -311,6 +312,8 @@ function BarRectangles(props: BarRectanglesProps) {
 
 const defaultMinPointSize: number = 0;
 
+const emptyArray: Array<never> = [];
```

Comment:
What's the reason for the empty arrays?

Reply:
So that the reference doesn't change between renders.